samples/nextjs/src/pages_examples/[[...path]].SSR.tsx

```diff
@@ -45,9 +45,13 @@ const SitecorePage = ({ notFound, layoutData, componentProps }: SitecorePageProp
 export const getServerSideProps: GetServerSideProps = async (context) => {
   const props = await sitecorePagePropsFactory.create(context);
 
+  // Returns custom 404 page with a status code of 404 when notFound: true
+  // Note we can't simply return props.notFound due to an issue in Next.js (https://github.com/vercel/next.js/issues/22472)
+  const notFound = props.notFound ? { notFound: true } : {};
+
   return {
     props,
-    notFound: props.notFound, // Returns custom 404 page with a status code of 404
```

Comment:
@ambrauer notFound: props.notFound || undefined

Reply:
@illiakovalenko Unfortunately, no. It appears the code is simply checking for the existence of the `notFound` property.

Reply:
The good news is that this will be resolved in the next release: https://github.com/vercel/next.js/pull/19861